src/Build/Resources/Strings.resx

```diff
@@ -331,6 +331,18 @@
   </data>
   <data name="TryingExtensionsPath" xml:space="preserve">
     <value>Trying to import {0} using extensions path {1}</value>
+  </data>
+    <data name="OverrideUsingTaskElementCreated" xml:space="preserve">
+    <value>MSB4275: Created an override using task: {0}</value>
```

Comment:
Do we need _both_ this and the "chosen" one? I feel like one or the other would do.

Reply:
Ping on this question.

Reply:
At first I figured why not have both, but notifying the build every time we picked a specific task doesn't provide much value. Especially when we can see the binary the task came from in the logs.

I'll remove the logic/resource for logging a message when an overridden task is chosen.